apps/meteor/server/models/raw/Uploads.ts

```diff
@@ -20,7 +20,7 @@ export class UploadsRaw extends BaseRaw<IUpload> implements IUploadsModel {
 	}
 
 	protected modelIndexes(): IndexDescription[] {
-		return [{ key: { rid: 1 } }, { key: { uploadedAt: 1 } }, { key: { typeGroup: 1 } }];
+		return [{ key: { rid: 1 } }, { key: { uploadedAt: 1 } }, { key: { typeGroup: 1 } }, { key: { _hidden: 1 } }];
```

Comment:
a better index would be a compound with `rid` and `_hidden`, since both are used together.
```suggestion
		return [{ key: { rid: 1, _hidden: 1 } }, { key: { uploadedAt: 1 } }, { key: { typeGroup: 1 } }];
```